ui/selectors/transactions.js

```diff
@@ -303,8 +303,7 @@ export const nonceSortedTransactionsSelector = createSelector(
           // known outside of the user's local MetaMask and the nonce
           // associated will be applied to the next.
           isEphemeral:
-            status === TRANSACTION_STATUSES.FAILED &&
-            txReceipt?.status !== '0x0',
+            status === TransactionStatus.failed && txReceipt?.status !== '0x0',
```

Comment:
we probably don't have a type for this yet, but just flagging it just in case. Does it make sense to put this in a constant?

Reply:
We should explore that. It would be really good to have a strict literal string type for 0x0.